client/src/components/Specification/Specification.jsx

```diff
@@ -16,11 +16,36 @@ const GETSPECIFICATION = gql`
 			requirements {
 				name
 				id
+				section {
+					name
+				}
 			}
 		}
 	}
 `;
 
+function sortRequirements(requirements) {
+	let sections = [];
+
+	for (let requirement of requirements) {
+		const section = requirement.section.name;
+		sections.push(section);
+	}
+
+	let uniqueSections = [...new Set(sections)];
+
+	let sortedRequirements = [];
+
+	for (let section of uniqueSections) {
```

Comment:
Same here can just use map e.g. sortedRequirements = unique sections.map()

You could prob do it all in one go over?